weasyl/controllers/api.py

```diff
@@ -152,6 +152,23 @@ def tidy_submission(submission):
     return submission
 
 
+def get_tidy_journal(journal):
+    new_journal = {x: journal[x] for x in journal if x in ["title", "content", "journalid", "tags"]}
+    new_journal['welcomeid'] = journal['id']
+    new_journal['posted_at'] = d.iso8601(journal['unixtime'])
+    contype = journal['contype'] if 'contype' in journal else None
+    if contype:
+        new_journal['type'] = m.CONTYPE_PARSABLE_MAP[contype]
+    new_journal['rating'] = ratings.CODE_TO_NAME[journal['rating']]
+    new_journal['owner'] = journal['username']
+    new_journal['owner_login'] = d.get_sysname(journal['username'])
+    journalid = journal['journalid']
+    if journalid > 0:
```

Comment:
When is a journalid ≤ 0?

Reply:
After reviewing `get_tidy_journal` and comparing it to `tidy_submission`, I conclude that I was overly defensive when I wrote it.